### PR TITLE
#1-Project-skeleton

### PR DESCRIPTION
**Pull request by BarTracz**
This should close #5

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+/.nb-gradle/
+build/
+!**/src/main/**/build/
+!**/src/test/**/build/
+
+### VS Code ###
+.vscode/
+
+### Mac OS ###
+.DS_Store
```
Comment (AleksandraKozubal) [inline .gitignore:38]: Please add empty line at the end of this and other files

**Review by AleksandraKozubal — COMMENTED** [review]
It would be nice to have some tips in readme.md on how to start it.

File: .gitignore (hunk 1 above)
Comment (BarTracz) [reply .gitignore:39]: Added

**Review by Lee0z — DISMISSED** [review]

File: readme.md
Comment (AleksandraKozubal) [inline readme.md:1]: Add some description about initial requirements like having java installed locally (which version if needed)

Comment (BarTracz) [reply readme.md:1]: readme updated with java requirements

**Review by AleksandraKozubal — APPROVED** [review]

**Review by Lee0z — APPROVED** [review]

**Pull request merged**